site/source/learn/concepts/networks.rst

```diff
@@ -16,17 +16,17 @@ A Local Area Network (LAN) is a computer network that interconnects computers wi
 
 Devices on a LAN are private and protected, such that only devices connected to the same Ethernet or WiFi network can see or communicate with them.
 
-Your Embassy hosts itself on the LAN and is reachable by visiting its ``embassy.local`` URL in the browser while also connected to the LAN.
+Your Embassy is hosted on the LAN and is reachable by visiting its ``embassy.local`` URL (for setup, and ``embassy-xxxxxxxx.local`` after setup) in the browser while also connected to the LAN.
 
 .. note:: Any device connected to a LAN can inspect all communications on that LAN. To avoid snooping, your Embassy's LAN communications are encrypted using :ref:`ssl`, which requires :ref:`additional setup <connecting-lan>`.
 
 .. _lan-cert:
 
 LAN Certificate
 ---------------
-When you visit an SSL secured website (https://), in addition to the connection being secured using SSL/TSL encryption, the website will present your browser with an SSL certificate showing that it (or more accurately ownership of the website's public key) has been authenticated by a recognized Certificate Authority (CA).  In the case of your Embassy, this certificate is "self-signed," and as such, the CA is you.
+When you visit an SSL website (https://), the connection is secured using SSL/TSL encryption. The website will present your browser with an SSL certificate showing that the owner of the website has been authenticated by a recognized Certificate Authority (CA). In the case of your Embassy, you are the CA and this certificate is self-signed!
```

Comment:
"Self-signed" means your cert was used as the message to be signed to produce a signature that is also attached to the cert.  When a CA (just another kind of public key cert with some special fields) signs your cert, since there are 2 different certs, one signing the other, it's technically not "self-signed" even though you made both certs.